src/foam/u2/wizard/data/CreateLoader.js

```diff
@@ -19,7 +19,7 @@ foam.CLASS({
       name: 'spec',
       factory: function(){
         return {
-          class: this.wizardletOf.id
+          class: this.wizardletOf?.id
```

Comment:
Do we know why this happens?

Reply:
It was during playing with the setup, and this error was blocking me from seeing the real issue.